WooCommerce/Classes/POS/Presentation/Settings/POSSettingsLocalCatalogDetailView.swift

```diff
@@ -0,0 +1,192 @@
+import SwiftUI
+
+struct POSSettingsLocalCatalogDetailView: View {
+    // TODO: WOOMOB-1335 - implement full sync cellular data setting functionality
+    @State private var allowFullSyncOnCellular: Bool = true
+
+    private var backgroundColor: Color {
+        Color.posOnSecondaryContainer
+    }
+
+    var body: some View {
+        NavigationStack {
+            VStack(spacing: POSSpacing.none) {
+                POSPageHeaderView(title: Localization.localCatalogTitle)
+                    .foregroundColor(.posSurface)
+                    .accessibilityAddTraits(.isHeader)
+
+                ScrollView {
+                    VStack(spacing: POSSpacing.medium) {
+                        catalogStatusView
+                        managingDataUsageView
+                        manualCatalogUpdateView
```

Comment:
```suggestion
                        catalogStatus
                        managingDataUsage
                        manualCatalogUpdate
```

They're not full views anyway, but even if they were, the word "view" seems unnecessary here

Reply:
Makes sense, renamed in https://github.com/woocommerce/woocommerce-ios/pull/16125/commits/3c4cac211d699311b6d748e4f556ec74d05c1368.